ui/src/App.tsx

```diff
@@ -1,16 +1,42 @@
 import * as React from 'react';
+import { Backend, StateRequest, ProxyState } from '../generated/web_pb';
+import { ProxyClient, ServiceClientOptions } from '../generated/web_pb_service';
 import './App.css';
 
 // import logo from './logo.svg';
 
+
```

Comment:
Here we make use of our generated client and request objects. We call the "state" method, which reports the proxy's state... when it works.